tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 # Tasks file for geonode
 
+- fail:
```

Comment:
They are set in `defaults`, so it won't fail.  It'll just use `admin/geoserver`, etc.